version.php

```diff
@@ -26,11 +26,11 @@
 
 /** @var stdClass $plugin */
 $plugin->component = 'filter_ally';
-$plugin->release   = '4.1.0';
-$plugin->version   = 2023061500;
+$plugin->release   = '4.2.0';
+$plugin->version   = 2023111500;
 $plugin->requires  = 2022041900;
 $plugin->maturity  = MATURITY_STABLE;
 $plugin->dependencies = [
-    'tool_ally'      => 2023041100,
-    'report_allylti' => 2023041100,
+    'tool_ally'      => 2023111500,
+    'report_allylti' => 2023111500,
```

Comment:
these versions don't exist @gthomas2 causing current master to fail to install. 

Was this really necessary?

Reply:
The versions do exist - there were other PRs bumping the versions in tool_ally and report_allylti but the PRs sat around for a while.